packages/browser/src/transports/fetch.ts

```diff
@@ -12,19 +12,31 @@ export class FetchTransport extends BaseTransport {
    * @inheritDoc
    */
   public sendEvent(event: Event): PromiseLike<Response> {
-    const eventType = event.type || 'event';
+    return this._sendRequest(eventToSentryRequest(event, this._api), event);
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public sendSession(session: Session): PromiseLike<Response> {
+    return this._sendRequest(sessionToSentryRequest(session, this._api), session);
+  }
 
-    if (this._isRateLimited(eventType)) {
+  /**
+   * @param sentryRequest Prepared SentryRequest to be delivered
+   * @param event original payload used to create SentryRequest
+   */
+  private _sendRequest(sentryRequest: SentryRequest, event: Event | Session): PromiseLike<Response> {
```

Comment:
`event: Event | Session` -- would be nice to have a more generic name to avoid confusion here.

```suggestion
   * @param originalPayload Original payload used to create SentryRequest
   */
  private _sendRequest(sentryRequest: SentryRequest, originalPayload: Event | Session): PromiseLike<Response> {
```